torch/_inductor/select_algorithm.py

```diff
@@ -117,6 +117,7 @@ def jit_line(self):
             "device": V.graph.scheduler.current_device.index,
             "device_type": V.graph.scheduler.current_device.type,
             "constants": {},
+            "origin_ops": {},
```

Comment:
Not sure if I should just remove it or add in TODO like you suggested.  I don't know what a **template** or **foreach** triton kernel look like.  Or more importantly, do you have an example of network code that would cause these kinds of kernels to be generated?

Reply:
I just went ahead and removed it.